### ISSUE TITLE
Why, we cant crop images?

### ISSUE THREAD

**Issue by omgbbqhaxx** [sixoverground/ImagePickerController#1]
your library is amazing. but we cant crop images can you add this feature?

**Comment by craigphares** [comment]
Yes, image cropping and filters will be added in the near future. Glad you like what's been done so far.

**Comment by omgbbqhaxx** [comment]
okey, im waiting new version.
